src/Microsoft.DotNet.Build.Tasks/PackageFiles/CodeCoverage.targets

```diff
@@ -131,6 +91,38 @@
   </Target>
   <!-- *********************************************************************************************** -->
 
+  <Target Name="CreateCoverageDedicatedRuntime"
+    BeforeTargets="GenerateTestExecutionScripts"
+    DependsOnTargets="GenerateWindowsPdbsForAssemblyBeingTested"
+    Condition="'$(CodeCoverageEnabled)'=='true' and '$(UseCoverageDedicatedRuntime)'=='true'">
+    <PropertyGroup>
+      <CoverageDedicatedRuntimeDir>$(TestHostRootPath)shared/Microsoft.NETCore.App/10.10.10</CoverageDedicatedRuntimeDir>
+    </PropertyGroup>
+    <ItemGroup>
+      <RuntimeFiles Include="$(NETCoreAppTestSharedFrameworkPath)/*.*" />
+      <RuntimeFiles Include="$(NETCoreAppTestSharedFrameworkPath)il/*.*" />
+      <RuntimeFilesAmd64 Include="$(NETCoreAppTestSharedFrameworkPath)amd64/*.*" />
+      <RuntimeFilesx86 Include="$(NETCoreAppTestSharedFrameworkPath)x86/*.*" />
+      <RuntimeWindowsPdbs Include="$(NETCoreAppTestSharedFrameworkPath)WindowsPDB/*.*" />
```

Comment:
What is putting the WindowsPDB directory here? If it is only being created for coverage can we start putting it somewhere else instead?

Reply:
That's a good point: we could start to have it only on the coverage folder. Let me see what I can do about it.